pkgs/applications/misc/joshuto/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchFromGitHub, lib, rustPlatform, stdenv, SystemConfiguration }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "joshuto";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kamiyaa";
+    repo = pname;
+    rev = version;
+    sha256 = "08d6h7xwcgycw5bdzwwc6aaikcrw3yc7inkiydgml9q261kql7zl";
+    # upstream includes an outdated Cargo.lock that stops cargo from compiling
+    postFetch = ''
```

Comment:
Move this into postPatch. postFetch is normally only used to fix issues with the zip eg on case insensitive fs'.